net/upnp/Makefile

```
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		upnp
-PLUGIN_VERSION=		1.6
+PLUGIN_VERSION=		1.7
 PLUGIN_REVISION=	1
```

```suggestion
PLUGIN_VERSION=		1.7
```

clear revision on new version